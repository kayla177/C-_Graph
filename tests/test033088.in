ENTITY B0Z1QJSH name_B0Z1 paper
ENTITY 1M1RZPX9 name_1M1R institution
ENTITY JGCCAKL7 name_JGCC conference
ENTITY DUUI4LOB name_DUUI journal
ENTITY 2OI12HSK name_2OI1 author
ENTITY L88P0XX9 name_L88P author
ENTITY PIZS716T name_PIZS paper
ENTITY I326F37M name_I326 paper
ENTITY TJDTSETK name_TJDT author
ENTITY S5HZI4QW name_S5HZ author
RELATIONSHIP 2OI12HSK affiliated_with PIZS716T 2.4
RELATIONSHIP B0Z1QJSH collaborated_with I326F37M 4.9
RELATIONSHIP I326F37M presented_at PIZS716T 7.9
RELATIONSHIP TJDTSETK cited_by L88P0XX9 9.2
RELATIONSHIP S5HZI4QW affiliated_with TJDTSETK 8.7
RELATIONSHIP L88P0XX9 published_in PIZS716T 9.5
RELATIONSHIP DUUI4LOB authored_by TJDTSETK 5.7
RELATIONSHIP TJDTSETK authored_by 1M1RZPX9 4.3
RELATIONSHIP 1M1RZPX9 collaborated_with 2OI12HSK 5.5
RELATIONSHIP JGCCAKL7 cited_by B0Z1QJSH 5.2
RELATIONSHIP S5HZI4QW published_in L88P0XX9 5.5
RELATIONSHIP S5HZI4QW collaborated_with PIZS716T 7.0
RELATIONSHIP DUUI4LOB cited_by S5HZI4QW 8.0
RELATIONSHIP 2OI12HSK published_in JGCCAKL7 5.0
RELATIONSHIP S5HZI4QW collaborated_with I326F37M 5.3
DELETE S5HZI4QW
DELETE 2OI12HSK
PRINT I326F37M
PRINT DUUI4LOB
PRINT B0Z1QJSH
DELETE JGCCAKL7
DELETE I326F37M
PRINT B0Z1QJSH
PATH TJDTSETK PIZS716T
DELETE 1M1RZPX9
EXIT
