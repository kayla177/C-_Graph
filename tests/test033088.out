success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
B0Z1QJSH PIZS716T
TJDTSETK
I326F37M JGCCAKL7
success
success

TJDTSETK L88P0XX9 PIZS716T 18.7
success
